include/NAS2D/Resources/Image.h

```diff
@@ -64,8 +64,8 @@ class Image: public Resource
 	void load();
 
 private:
-	std::pair<int, int>				_size;		/**< Width/Height information about the Image. */
-	std::pair<uint32_t, uint32_t>	_center;	/**<  */
+    std::pair<int, int> _size{0,0};		   /**< Width/Height information about the Image. */
+    std::pair<uint32_t, uint32_t> _center{0u,0u}; /**<  */
```

Comment:
Kind of off topic, though we should probably avoid using `std::pair` (or tuples) outside of template code. That's a separate issue though. Just noting it down so I don't forget.

Reply:
I was refactoring the Image class to #391 specs (I've since decided to move that refactor back a bit behind others) and the first thing I did was replace these with `Vector<int>` and `Vector<uint32_t>` respectively. So I'm in favor of not using `std::pair`. Further off topic, why is center a `uint32_t` but size is an `int`? Maybe that should be normalized too.

Reply:
I think Leeor was using `uint32_t` as a short hand for `unsigned`. Unfortunately that doesn't play so nice with the differences between 32-bit and 64-bit.

I agree with replacing with `Vector<int>`.

We can probably remove `center`. The `Vector` class supports scalar division, so it's really easy to do without a special method. It might be worth keeping though if we plan to someday support off center images, where the logical center of the object is not the center of the sprite image.

Reply:
Opened issue #399 to track.